catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -200,14 +211,14 @@ private void executeRemoveAllFromStore() throws Exception {
 
     String info =
         String.format(
-            " %d file(s) removed in %3.3f seconds%n",
+            "%d file(s) removed in %3.3f seconds",
             totalAmountDeleted, (end - start) / MS_PER_SECOND);
 
     LOGGER.info(info);
-    LOGGER.info(totalAmountDeleted + " files removed using cache:removeAll command");
+    LOGGER.info("{} file(s) removed using the \"{}\" command", totalAmountDeleted, COMMAND);
 
     console.println();
-    console.print(info);
+    console.println(info);
```

Comment:
Update output:
```
admin@root()> catalog:removeall
WARNING: This will permanently remove all records from the Catalog. Do you want to proceed? (yes/no): yes

0 file(s) removed in 0.012 seconds
admin@root()> log:tail
09:01:36,898 | INFO  | atalog:removeall | ce.ddf.commands.catalog.RemoveAllCommand  212 | atalog-core-commands | 0 file(s) removed in 0.012 seconds
09:01:36,898 | INFO  | atalog:removeall | ce.ddf.commands.catalog.RemoveAllCommand  213 | atalog-core-commands | 0 files removed using the "catalog:removeall" command
```